cdk/database/__init__.py

```diff
@@ -9,9 +9,14 @@ class Database(core.Stack):
     def __init__(self, scope: core.Construct,
                  stage: str, *, env: core.Environment):
         self.id = f'Database-{stage}'
-        super().__init__(scope, self.id, env=env, termination_protection=True)
+        self.users_id = f'-users-{stage}'
+        self.visits_id = f'-visits-{stage}'
+        super().__init__(
+            scope, f'Database-{stage}', env=env, termination_protection=True)
```

Comment:
I suspect that because you've replaced the `self.id` here the entire Database Stack is getting erased - perhaps that is what causes the dynamodb single table to get replaced?